### ISSUE TITLE
Express Checkout - No Line Items

### ISSUE THREAD

**Issue by angelleye** [angelleye/paypal-woocommerce#330]
Seems that when the option to include line items with Express Checkout is disabled, then we aren't even getting any order total to show up at PayPal.  Here are details from a user report.

>With the "Send line item details to Paypal" not checked:

>Paypal Express Checkout options page

>![image](https://cloud.githubusercontent.com/assets/629241/10436767/12fb20f4-70ee-11e5-902c-10ad40417e49.png)

>Cart and order are the same in the previous screenshots

>But paypal page is this:

>![image](https://cloud.githubusercontent.com/assets/629241/10436775/2176f202-70ee-11e5-917d-82dd921aedbc.png)

>There is no description of each item (and that it would be ok) but there is not also the main description of the order!

We need to look into this because I think we should be seeing the order total here and maybe a basic description.  Need to check what parameters we're sending when line items aren't included.

**Comment by levansy2020** [comment]
Result here:
http://screencast.com/t/YVUNvAl5DOnv

I only put the text Order there, if you need to change to anything please comment here.

**Comment by angelleye** [comment]
Actually, that is not what I was wanting to do here.  

I was thinking there must be some parameter we were leaving out which caused the total to not show up on the PayPal checkout pages even without any line items at all.  Upon further review I see that is not the case.  Even when we include the AMT parameter as expected, it does not display on the PayPal checkout page.

What I did find, though, is that if I set useraction=commit in the redirect URL (which happens when we enable the "skip final review" option in the plugin) then it does show the order total on the PayPal screen.

I think I'm going to leave this update out of the plugin because it's not really doing what was intended.  I'll just let the user know they need to enable the skip review option if they want the order to show up there.  Otherwise, the PayPal systems assumes you'll be displaying order details in your own review screen, which we would be, of course.
